requirements.txt

```diff
@@ -38,6 +38,8 @@ jsonref==0.2
     # via flattentool
 lxml==4.6.3
     # via flattentool
+millify==0.1.1
+    # via -r requirements.in
```

Comment:
Could we just use the "humanize" django module for? -https://docs.djangoproject.com/en/3.2/ref/contrib/humanize/

Reply:
Ah I see that it's actually doing the 10k etc